### ISSUE TITLE
Stryker breaks the jest environment change and tests fail

### ISSUE THREAD

**Issue by eggei** [stryker-mutator/stryker#2569]
Hey there,

Eventhough the jest runs smoothly, I get this error when I run `stryker run`

```
13:22:15 (8356) ERROR DryRunExecutor One or more tests failed in the initial test run:
        Tests For "policyListOfTopFailure" State API Call getPolicyStatsAPI fetches the data and returns it as expected
                ReferenceError: Headers is not defined
    at callAPI$ (/Users/....../dashboard-ui/.stryker-tmp/sandbox9622414/src/shared/util/api-service.ts:100:11)
.
.
.
```
I have this statement in the file where this error is being thrown:
```
/**
 * @jest-environment jsdom
 */
```

But I can see stryker places another comment in this file in the created sandbox which looks like this:
```
// @ts-nocheck
/**
 * @jest-environment jsdom
 */
```
and it breaks the functionality of the decorator comes after it.

**Stryker config**

```javascript
/**
 * @type {import('@stryker-mutator/api/core').StrykerOptions}
 */
module.exports = {
  packageManager: 'npm',
  reporters: ['html', 'clear-text', 'progress'],
  testRunner: 'jest',
  coverageAnalysis: 'off',
}

```

**Test runner config**

```javascript
module.exports = {
  roots: ['<rootDir>/src'],
  preset: 'ts-jest/presets/js-with-babel',
  globals: {
    'ts-jest': {
      babelConfig: true,
    },
  },
  verbose: true,
  testEnvironment: 'node',
  collectCoverageFrom: ['src/**/*.ts', '!src/**/index.ts', '!src/**/*.d.ts'],
  moduleNameMapper: {
    '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
      '<rootDir>/__mocks__/fileMock.js',
    '\\.(css|less)$': 'identity-obj-proxy',
  },
  testPathIgnorePatterns: ['mock.helpers.ts'],
  coveragePathIgnorePatterns: ['mock.helpers.ts'],
}
```

**Stryker environment**
```json
"@stryker-mutator/core": "^4.0.0",
"@stryker-mutator/jest-runner": "^4.0.0",
```

```json
"jest": "^25.5.4"
"@types/jest": "^25.2.1"
"ts-jest": "^25.5.1"
```

**Test runner environment**
```shell
$ jest
```

**Your Environment**

| software         | version(s)
| ---------------- | -------
| node             | 14.12.0
| npm              | 6.14.8
| Operating System | MacOS Catalina 10.15.7 (19H2)

**Add stryker.log**
(I have to paste it because it is restricted to upload a file on the computer I am working)
<details>
<summary>See stryker.log</summary>
<br>

```
13:44:37 (10061) INFO ConfigReader Using stryker.conf.js
13:44:37 (10061) DEBUG BroadcastReporter Broadcasting to reporters ["html","clear-text","progress"]
<DELETED LOGS FOR SECURITY REASONS>
13:44:37 (10061) DEBUG ConfigReader Loaded config: {
  "packageManager": "npm",
  "reporters": [
    "html",
    "clear-text",
    "progress"
  ],
  "testRunner": "jest",
  "coverageAnalysis": "off",
  "allowConsoleColors": true,
  "checkers": [],
  "maxTestRunnerReuse": 0,
  "commandRunner": {
    "command": "npm test"
  },
  "clearTextReporter": {
    "allowColor": true,
    "logTests": true,
    "maxTestsToLog": 3
  },
  "dashboard": {
    "baseUrl": "https://dashboard.stryker-mutator.io/api/reports",
    "reportType": "full"
  },
  "eventReporter": {
    "baseDir": "reports/mutation/events"
  },
  "fileLogLevel": "trace",
  "logLevel": "info",
  "maxConcurrentTestRunners": 9007199254740991,
  "mutate": [
    "{src,lib}/**/!(*.+(s|S)pec|*.+(t|T)est).+(cjs|mjs|js|ts|jsx|tsx|html|vue)",
    "!{src,lib}/**/__tests__/**/*.+(cjs|mjs|js|ts|jsx|tsx|html|vue)"
  ],
  "mutator": {
    "plugins": null,
    "excludedMutations": []
  },
  "plugins": [
    "@stryker-mutator/*"
  ],
  "appendPlugins": [],
  "disableTypeChecks": "{test,src,lib}/**/*.{js,ts,jsx,tsx,html,vue}",
  "symlinkNodeModules": true,
  "tempDirName": ".stryker-tmp",
  "cleanTempDir": true,
  "thresholds": {
    "high": 80,
    "low": 60,
    "break": null
  },
  "timeoutFactor": 1.5,
  "timeoutMS": 5000,
  "tsconfigFile": "tsconfig.json",
  "warnings": true,
<DELETED LOGS FOR SECURITY REASONS>
}
<DELETED LOGS FOR SECURITY REASONS>
13:44:37 (10061) INFO InputFileResolver Found 106 of 193 file(s) to be mutated.
13:44:37 (10061) DEBUG InputFileResolver All input files: [
<DELETED LOGS FOR SECURITY REASONS>
]
13:44:37 (10061) DEBUG InputFileResolver Files to mutate: [
<DELETED LOGS FOR SECURITY REASONS>
]
<DELETED LOGS FOR SECURITY REASONS>
13:44:38 (10061) INFO Instrumenter Instrumented 106 source file(s) with 2423 mutant(s)
13:44:38 (10061) DEBUG TSConfigPreprocessor Rewriting file File {
  name: <DELETED LOGS FOR SECURITY REASONS>,
  _content: <Buffer 7b 0a 20 20 22 63 6f 6d 70 69 6c 65 72 4f 70 74 69 6f 6e 73 22 3a 20 7b 0a 20 20 20 20 22 6e 6f 45 6d 69 74 22 3a 20 74 72 75 65 20 2f 2a 20 42 65 63 ... 1625 more bytes>,
  _textContent: '{\n' +
    '  "compilerOptions": {\n' +
    '    "noEmit": true /* Because webpack handles the compilation, here, we tell tsc to just check for types but not to compile the code */,\n' +
    '    "esModuleInterop": true /* Emit __importStar and __importDefault helpers for runtime babel ecosystem compatibility and enable --allowSyntheticDefaultImports for typesystem compatibility. */,\n' +
    '    "experimentalDecorators": true /* Enables experimental support for ES decorators. */,\n' +
    '    "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */,\n' +
    '    "jsx": "react" /* Support JSX in .tsx files */,\n' +
    '    "moduleResolution": "node" /* Determine how modules get resolved. */,\n' +
    '    "noErrorTruncation": true /* Do not truncate error messages. */,\n' +
    '    "noUnusedLocals": true /* Report errors on unused locals. */,\n' +
    '    "noUnusedParameters": true /* Report errors on unused parameters. */,\n' +
    '    "skipLibCheck": true /* Skip type checking of all declaration files (*.d.ts). */,\n' +
    '    "strict": true /* Enable all strict type checking options. */,\n' +
    '    "target": "esnext" /* Specify ECMAScript target version: esnext targets the latest supported features */,\n' +
    '    "typeRoots": [\n' +
    '      "./node_modules/@types",\n' +
    '      "src/shared/types"\n' +
    '    ] /* List of folders to include type definitions from.  */\n' +
    '  },\n' +
    '  "include": [\n' +
    '    "src/",\n' +
    '    "webpack.config.js",\n' +
    '    "stylelint.config.js",\n' +
    '    "prettier.config.js",\n' +
    '    "postcss.config.js",\n' +
    '    "jest.config.js",\n' +
    '    "stryker.conf.js",\n' +
    '    "__mocks__"\n' +
    '  ] /* Include files for typecheck */,\n' +
    '  "exclude": ["types/"] /* Exclude files for typecheck */\n' +
    '}\n' +
    '\n' +
    '// For more info: https://www.typescriptlang.org/docs/handbook/compiler-options.html\n'
}
13:44:38 (10061) INFO ConcurrencyTokenProvider Creating 11 test runner process(es).
<DELETED LOGS FOR SECURITY REASONS>
13:44:40 (10061) INFO DryRunExecutor Starting initial test run. This may take a while.
13:44:40 (10061) DEBUG TimeoutDecorator Starting timeout timer (300000 ms) for a test run
<DELETED LOGS FOR SECURITY REASONS>
13:44:40 (10071) DEBUG JestTestRunner Merging file-based config {
  "roots": [
    "<rootDir>/src"
  ],
  "preset": "ts-jest/presets/js-with-babel",
  "globals": {
    "ts-jest": {
      "babelConfig": true
    }
  },
  "verbose": true,
  "testEnvironment": "node",
  "collectCoverageFrom": [
    "src/**/*.ts",
    "!src/**/index.ts",
    "!src/**/*.d.ts"
  ],
  "moduleNameMapper": {
    "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
    "\\.(css|less)$": "identity-obj-proxy"
  },
  "testPathIgnorePatterns": [
    "mock.helpers.ts"
  ],
  "coveragePathIgnorePatterns": [
    "mock.helpers.ts"
  ]
} 
      with custom config {}
      and default (internal) stryker config [object Object]
13:44:40 (10071) DEBUG JestTestRunner Running jest with --findRelatedTests flag. Set jest.enableFindRelatedTests to false to run all tests on every mutant.
<DELETED LOGS FOR SECURITY REASONS>
13:44:40 (10071) TRACE JestGreaterThan25Adapter Invoking Jest with config {"roots":["<rootDir>/src"],"preset":"ts-jest/presets/js-with-babel","globals":{"ts-jest":{"babelConfig":true}},"verbose":false,"testEnvironment":"node","collectCoverageFrom":["src/**/*.ts","!src/**/index.ts","!src/**/*.d.ts"],"moduleNameMapper":{"\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$":"<rootDir>/__mocks__/fileMock.js","\\.(css|less)$":"identity-obj-proxy"},"testPathIgnorePatterns":["mock.helpers.ts"],"coveragePathIgnorePatterns":["mock.helpers.ts"],"collectCoverage":false,"bail":false,"notify":false,"rootDir":"<DELETED LOGS FOR SECURITY REASONS>","reporters":[]}
<DELETED LOGS FOR SECURITY REASONS>
<DELETED LOGS FOR SECURITY REASONS>
13:44:40 (10069) DEBUG JestTestRunner Merging file-based config {
  "roots": [
    "<rootDir>/src"
  ],
  "preset": "ts-jest/presets/js-with-babel",
  "globals": {
    "ts-jest": {
      "babelConfig": true
    }
  },
  "verbose": true,
  "testEnvironment": "node",
  "collectCoverageFrom": [
    "src/**/*.ts",
    "!src/**/index.ts",
    "!src/**/*.d.ts"
  ],
  "moduleNameMapper": {
    "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
    "\\.(css|less)$": "identity-obj-proxy"
  },
  "testPathIgnorePatterns": [
    "mock.helpers.ts"
  ],
  "coveragePathIgnorePatterns": [
    "mock.helpers.ts"
  ]
} 
      with custom config {}
      and default (internal) stryker config [object Object]
13:44:40 (10069) DEBUG JestTestRunner Running jest with --findRelatedTests flag. Set jest.enableFindRelatedTests to false to run all tests on every mutant.
<DELETED LOGS FOR SECURITY REASONS>
13:44:41 (10160) DEBUG JestTestRunner Merging file-based config {
  "roots": [
    "<rootDir>/src"
  ],
  "preset": "ts-jest/presets/js-with-babel",
  "globals": {
    "ts-jest": {
      "babelConfig": true
    }
  },
  "verbose": true,
  "testEnvironment": "node",
  "collectCoverageFrom": [
    "src/**/*.ts",
    "!src/**/index.ts",
    "!src/**/*.d.ts"
  ],
  "moduleNameMapper": {
    "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
    "\\.(css|less)$": "identity-obj-proxy"
  },
  "testPathIgnorePatterns": [
    "mock.helpers.ts"
  ],
  "coveragePathIgnorePatterns": [
    "mock.helpers.ts"
  ]
} 
      with custom config {}
      and default (internal) stryker config [object Object]
13:44:41 (10160) DEBUG JestTestRunner Running jest with --findRelatedTests flag. Set jest.enableFindRelatedTests to false to run all tests on every mutant.
<DELETED LOGS FOR SECURITY REASONS>
13:44:41 (10161) DEBUG JestTestRunner Merging file-based config {
  "roots": [
    "<rootDir>/src"
  ],
  "preset": "ts-jest/presets/js-with-babel",
  "globals": {
    "ts-jest": {
      "babelConfig": true
    }
  },
  "verbose": true,
  "testEnvironment": "node",
  "collectCoverageFrom": [
    "src/**/*.ts",
    "!src/**/index.ts",
    "!src/**/*.d.ts"
  ],
  "moduleNameMapper": {
    "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
    "\\.(css|less)$": "identity-obj-proxy"
  },
  "testPathIgnorePatterns": [
    "mock.helpers.ts"
  ],
  "coveragePathIgnorePatterns": [
    "mock.helpers.ts"
  ]
} 
      with custom config {}
      and default (internal) stryker config [object Object]
13:44:41 (10161) DEBUG JestTestRunner Running jest with --findRelatedTests flag. Set jest.enableFindRelatedTests to false to run all tests on every mutant.
<DELETED LOGS FOR SECURITY REASONS>
  "roots": [
    "<rootDir>/src"
  ],
  "preset": "ts-jest/presets/js-with-babel",
  "globals": {
    "ts-jest": {
      "babelConfig": true
    }
  },
  "verbose": true,
  "testEnvironment": "node",
  "collectCoverageFrom": [
    "src/**/*.ts",
    "!src/**/index.ts",
    "!src/**/*.d.ts"
  ],
  "moduleNameMapper": {
    "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
    "\\.(css|less)$": "identity-obj-proxy"
  },
  "testPathIgnorePatterns": [
    "mock.helpers.ts"
  ],
  "coveragePathIgnorePatterns": [
    "mock.helpers.ts"
  ]
} 
      with custom config {}
      and default (internal) stryker config [object Object]
13:44:42 (10222) DEBUG JestTestRunner Running jest with --findRelatedTests flag. Set jest.enableFindRelatedTests to false to run all tests on every mutant.
<DELETED LOGS FOR SECURITY REASONS>
13:44:42 (10218) DEBUG JestTestRunner Merging file-based config {
  "roots": [
    "<rootDir>/src"
  ],
  "preset": "ts-jest/presets/js-with-babel",
  "globals": {
    "ts-jest": {
      "babelConfig": true
    }
  },
  "verbose": true,
  "testEnvironment": "node",
  "collectCoverageFrom": [
    "src/**/*.ts",
    "!src/**/index.ts",
    "!src/**/*.d.ts"
  ],
  "moduleNameMapper": {
    "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
    "\\.(css|less)$": "identity-obj-proxy"
  },
  "testPathIgnorePatterns": [
    "mock.helpers.ts"
  ],
  "coveragePathIgnorePatterns": [
    "mock.helpers.ts"
  ]
} 
      with custom config {}
      and default (internal) stryker config [object Object]
13:44:42 (10218) DEBUG JestTestRunner Running jest with --findRelatedTests flag. Set jest.enableFindRelatedTests to false to run all tests on every mutant.
<DELETED LOGS FOR SECURITY REASONS>
13:44:42 (10061) TRACE ChildProcessProxy ts-jest[versions] (WARN) Version 4.0.3 of typescript installed has not been tested with ts-jest. If you're experiencing issues, consider using a supported version (>=3.4.0 <4.0.0). Please do not report issues in ts-jest if you are using unsupported versions.

<DELETED LOGS FOR SECURITY REASONS>
13:44:43 (10293) DEBUG JestTestRunner Merging file-based config {
  "roots": [
    "<rootDir>/src"
  ],
  "preset": "ts-jest/presets/js-with-babel",
  "globals": {
    "ts-jest": {
      "babelConfig": true
    }
  },
  "verbose": true,
  "testEnvironment": "node",
  "collectCoverageFrom": [
    "src/**/*.ts",
    "!src/**/index.ts",
    "!src/**/*.d.ts"
  ],
  "moduleNameMapper": {
    "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
    "\\.(css|less)$": "identity-obj-proxy"
  },
  "testPathIgnorePatterns": [
    "mock.helpers.ts"
  ],
  "coveragePathIgnorePatterns": [
    "mock.helpers.ts"
  ]
} 
      with custom config {}
      and default (internal) stryker config [object Object]
13:44:43 (10293) DEBUG JestTestRunner Running jest with --findRelatedTests flag. Set jest.enableFindRelatedTests to false to run all tests on every mutant.
<DELETED LOGS FOR SECURITY REASONS>
13:44:43 (10291) DEBUG JestTestRunner Merging file-based config {
  "roots": [
    "<rootDir>/src"
  ],
  "preset": "ts-jest/presets/js-with-babel",
  "globals": {
    "ts-jest": {
      "babelConfig": true
    }
  },
  "verbose": true,
  "testEnvironment": "node",
  "collectCoverageFrom": [
    "src/**/*.ts",
    "!src/**/index.ts",
    "!src/**/*.d.ts"
  ],
  "moduleNameMapper": {
    "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
    "\\.(css|less)$": "identity-obj-proxy"
  },
  "testPathIgnorePatterns": [
    "mock.helpers.ts"
  ],
  "coveragePathIgnorePatterns": [
    "mock.helpers.ts"
  ]
} 
      with custom config {}
      and default (internal) stryker config [object Object]
13:44:43 (10291) DEBUG JestTestRunner Running jest with --findRelatedTests flag. Set jest.enableFindRelatedTests to false to run all tests on every mutant.
<DELETED LOGS FOR SECURITY REASONS>
13:44:43 (10334) DEBUG JestTestRunner Merging file-based config {
  "roots": [
    "<rootDir>/src"
  ],
  "preset": "ts-jest/presets/js-with-babel",
  "globals": {
    "ts-jest": {
      "babelConfig": true
    }
  },
  "verbose": true,
  "testEnvironment": "node",
  "collectCoverageFrom": [
    "src/**/*.ts",
    "!src/**/index.ts",
    "!src/**/*.d.ts"
  ],
  "moduleNameMapper": {
    "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
    "\\.(css|less)$": "identity-obj-proxy"
  },
  "testPathIgnorePatterns": [
    "mock.helpers.ts"
  ],
  "coveragePathIgnorePatterns": [
    "mock.helpers.ts"
  ]
} 
      with custom config {}
      and default (internal) stryker config [object Object]
13:44:43 (10334) DEBUG JestTestRunner Running jest with --findRelatedTests flag. Set jest.enableFindRelatedTests to false to run all tests on every mutant.
<DELETED LOGS FOR SECURITY REASONS>
13:44:43 (10335) DEBUG JestTestRunner Merging file-based config {
  "roots": [
    "<rootDir>/src"
  ],
  "preset": "ts-jest/presets/js-with-babel",
  "globals": {
    "ts-jest": {
      "babelConfig": true
    }
  },
  "verbose": true,
  "testEnvironment": "node",
  "collectCoverageFrom": [
    "src/**/*.ts",
    "!src/**/index.ts",
    "!src/**/*.d.ts"
  ],
  "moduleNameMapper": {
    "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
    "\\.(css|less)$": "identity-obj-proxy"
  },
  "testPathIgnorePatterns": [
    "mock.helpers.ts"
  ],
  "coveragePathIgnorePatterns": [
    "mock.helpers.ts"
  ]
} 
      with custom config {}
      and default (internal) stryker config [object Object]
13:44:43 (10335) DEBUG JestTestRunner Running jest with --findRelatedTests flag. Set jest.enableFindRelatedTests to false to run all tests on every mutant.
<DELETED LOGS FOR SECURITY REASONS>
13:44:44 (10336) DEBUG JestTestRunner Merging file-based config {
  "roots": [
    "<rootDir>/src"
  ],
  "preset": "ts-jest/presets/js-with-babel",
  "globals": {
    "ts-jest": {
      "babelConfig": true
    }
  },
  "verbose": true,
  "testEnvironment": "node",
  "collectCoverageFrom": [
    "src/**/*.ts",
    "!src/**/index.ts",
    "!src/**/*.d.ts"
  ],
  "moduleNameMapper": {
    "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
    "\\.(css|less)$": "identity-obj-proxy"
  },
  "testPathIgnorePatterns": [
    "mock.helpers.ts"
  ],
  "coveragePathIgnorePatterns": [
    "mock.helpers.ts"
  ]
} 
      with custom config {}
      and default (internal) stryker config [object Object]
13:44:44 (10336) DEBUG JestTestRunner Running jest with --findRelatedTests flag. Set jest.enableFindRelatedTests to false to run all tests on every mutant.
<DELETED LOGS FOR SECURITY REASONS>
13:44:51 (10061) ERROR DryRunExecutor One or more tests failed in the initial test run:
	Tests For "policyListOfTopFailure" State API Call getPolicyStatsAPI fetches the data and returns it as expected
		ReferenceError: Headers is not defined
    at callAPI$ (/Users/<DELETED LOGS FOR SECURITY REASONS>/dashboard-ui/.stryker-tmp/sandbox4090670/src/shared/util/api-service.ts:100:11)
    at tryCatch (/Users/<DELETED LOGS FOR SECURITY REASONS>/dashboard-ui/node_modules/regenerator-runtime/runtime.js:63:40)
    <DELETED LOGS FOR SECURITY REASONS>
	Tests For "policyListOfTopFailure" State API Call getPolicyStatsAPI returns error on exception
		Error: expect(received).rejects.toEqual(expected) // deep equality

<DELETED LOGS FOR SECURITY REASONS>
13:44:51 (10061) DEBUG Stryker Not removing the temp dir because an error occurred
13:44:51 (10061) ERROR Stryker There were failed tests in the initial test run.
13:44:51 (10061) DEBUG ChildProcessProxy Disposing of worker process 10071
13:44:51 (10061) DEBUG ChildProcessProxy Disposing of worker process 10069
13:44:51 (10061) DEBUG ChildProcessProxy Disposing of worker process 10160
13:44:51 (10061) DEBUG ChildProcessProxy Disposing of worker process 10161
13:44:51 (10061) DEBUG ChildProcessProxy Disposing of worker process 10218
13:44:51 (10061) DEBUG ChildProcessProxy Disposing of worker process 10293
13:44:51 (10061) DEBUG ChildProcessProxy Disposing of worker process 10222
13:44:51 (10061) DEBUG ChildProcessProxy Disposing of worker process 10291
13:44:51 (10061) DEBUG ChildProcessProxy Kill 10071
13:44:51 (10061) DEBUG ChildProcessProxy Disposing of worker process 10334
13:44:51 (10061) DEBUG ChildProcessProxy Disposing of worker process 10336
13:44:51 (10061) DEBUG ChildProcessProxy Disposing of worker process 10335
13:44:51 (10061) DEBUG ChildProcessProxy Kill 10069
13:44:51 (10061) DEBUG ChildProcessProxy Kill 10161
13:44:51 (10061) DEBUG ChildProcessProxy Kill 10160
13:44:51 (10061) DEBUG ChildProcessProxy Kill 10218
13:44:51 (10061) DEBUG ChildProcessProxy Kill 10293
13:44:51 (10061) DEBUG ChildProcessProxy Kill 10222
13:44:51 (10061) DEBUG ChildProcessProxy Kill 10291
13:44:51 (10061) DEBUG ChildProcessProxy Kill 10334
13:44:51 (10061) DEBUG ChildProcessProxy Kill 10335
13:44:51 (10061) DEBUG ChildProcessProxy Kill 10336

```
</details>

**Comment by nicojs** [comment]
Thanks for opening this issue and the detailed error report!

As a workaround, can you try not disabling typechecking?

```json
{
  "disableTypeChecks": false
}
```

It should work as long as you don't use `ts-jest`

**Comment by eggei** [comment]
Of course! 
Hmm, unfortunately I use `ts-jest`. I tried `disableTypeChecks` anyway but I got bunch of type errors.
I really wanted to try Stryker but I assume there is only one workaround you could recommend, no?

**Comment by eggei** [comment]
@nicojs Why doesn't it work if I use `ts-jest` is there something I can do to make it work with it?

**Comment by nicojs** [comment]
This is because `ts-jest` uses typescript to do type-checking:

> It supports all features of TypeScript including type-checking

The reason why type-checking doesn't work with Stryker is explained in our latest blog post: https://stryker-mutator.io/blog/announcing-stryker-4-mutation-switching/

I'll be working on a fix.

As a workaround: `"disableTypeChecks"` actually also excepts a glob expression. You could try to only target those specific files that needs their type checking disabled (thus files without ` @jest-environment jsdom`).

**Comment by nicojs** [comment]
Of course, you could also choose to not run Stryker for those test files.

**Comment by nicojs** [comment]
The fix is released in v4.1.1
